xmtp_mls/src/groups/mod.rs

```diff
@@ -521,16 +530,7 @@ impl MlsGroup {
         account_addresses_to_remove: Vec<InboxId>,
     ) -> Result<(), GroupError> {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_remove)?;
-        let conn = self.context.store.conn()?;
-        let intent_data: Vec<u8> = RemoveMembersIntentData::new(account_addresses.into()).into();
-
-        self.pre_intent_hook(client).await?;
-
-        let intent = conn.insert_group_intent(NewGroupIntent::new(
-            IntentKind::RemoveMembers,
-            self.group_id.clone(),
-            intent_data,
-        ))?;
+        let inbox_id_map = client.api_client.get_inbox_ids(account_addresses).await?;
```

Comment:
@richardhuaaa Also the same question here - we don't have new group intent anymore, if I should apply the `pre_intent_hook` and where is the best place to apply it?

Reply:
See above answer!